GETTING STARTED - Developers.rdoc

```diff
@@ -0,0 +1,61 @@
+
+It's likely that many folks reading this are Ruby-on-Rails developers.  Wagn owes a great deal of debt to the Rails community, both in terms of code and in stylistic guidance.  Rails developers will find lots of familiar patterns in Wagn, but we should be clear: <b>Wagn development is different</b>.
+
+Typical Rails development will involve creating lots of new models, views, and controllers. Wagn mod developers do create lots of views (though they're quite different from Rails views), but we never create new models or controllers.  In the Wagn gem there is one controller ({CardController}), one main model ({Card}), and several other models to optimize the Card model and track its history ({Card::Reference}, {Card::Act}, {Card::Action}, and {Card::Change}).
+
+Most Wagn developers never have to give the controller or any model besides Card any thought. The equivalent of creating a new model is configuring a new Set of cards. The equivalent of creating new actions on controllers is creating new events that are conditionally triggeres for a Set of cards.
+
+So, as a starting place it's probably <b>best to think of Wagn not as a Rails app but a Rails descendant</b>. Rails development experience is definitely a plus in getting to know Wagn, but Wagn offers a new architecture, and lot of new possibilities.
+
+
+==MoFoS Architecture
+
+Wagn is not, like Rails, an implementation of the "MVC" (Model-View-Controller) architectural pattern, or any well-established pattern, really.  Instead, we like to think that we're the first instance of a new pattern that we call _MoFoS_.
+
+MoFoS stands for "Model-Formats-Sets". It's intentional that the word "Model" alone is singular; the idea is that there is just one primary data model (Card, in Wagn's case), and that this model is given meaning by subdividing into multiple configurable Sets and made productive by connecting cards to multiplfe Formats.
+
+_Sets_ are a central organizing concept for both Wagneers and Wagn developers. If you've edited Rules through a card toolbar in any Wagn deck, you will have encountered Sets. For example, imagine you're on a User card named "Henry Tai", and you decide to edit its structure.  When you edit its rule, you will be prompted to choose the Set of cards to which the rule applies, eg:
+
+* Just "Henry Tai"
+* All Users
+* All Cards
+
+Depending on which Set you choose, the Rule in question will be applied to a specific group of cards.  This same general pattern occurs in Wagn code: <b>code rules (methods) are organized under the Set of cards to which they apply.</b>
+
+The great power of MoFoS is that, unlike in MVC, code rules (and data rules) can be as narrow or specific as desired.  In MVC, it's challenging to create rules that apply to multiple models or single instances of a model; in MoFoS, this is very fluid.
+
+
+==Mods
+
+A Wagn Mod is a discrete piece of Wagn functionality
+
+chunk       format      layout      set         set_pattern spec  file  lib
+
```

Comment:
it's a stub.

I'm not sure I'm happy with the organization of this doc, but mostly because it does too much "general to specific".  It needs to start with more examples and then work towards general ideas; that's really how people learn.